### ISSUE TITLE
WMS configuration error

### ISSUE THREAD

**Issue by floran974** [jancelin/docker-lizmap#29]
Hi !

I  was install docker-lizmap i my ubuntu server and i have his error when i  load the map

`<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">
 <ServiceException code="WMS configuration error">There was an error reading the project file or the SLD configuration</ServiceException>
</ServiceExceptionReport>
`

i use  QGIS 2.14.3-Essen

Thank for your help

**Comment by jancelin** [comment]
Hi,
change URL WMS on back-office (http://ip/websig/lizmap/www/admin.php):
You have to change the localhost to qgisserver:
like that:

```
        http://qgiserver/cgi-bin/qgis_mapserv.fcgi
```

**Comment by floran974** [comment]
yes , but i have this error with Lizmap generic configuration fro the Cache root directory*

> The directory does not exists or is not writable. You can use: /tmp
